python-avd/pyavd/_eos_designs/structured_config/base/platform_mixin.py

```diff
@@ -0,0 +1,132 @@
+
+    @cached_property
+    def _get_sfe_interface_profile_member_interfaces(
+        self: AvdStructuredConfigBaseProtocol,
+    ) -> EosCliConfigGen.Platform.Sfe.Interface.ProfilesItem.Interfaces | None:
+        """Returns list of eligible SFE interfaces with profile settings for structured config."""
+        # Iterate through all L3 interfaces checking for those with 'rx_queue' config
+        # Also iterate through member interfaces of all L3 Port-Channels with 'rx_queue' config
+        if not self.shared_utils.is_sfe_interface_profile_supported:
+            return None
+        sfe_profile_member_interfaces = EosCliConfigGen.Platform.Sfe.Interface.ProfilesItem.Interfaces()
+        # iterate through each L3 interface
+        for interface in self.shared_utils.l3_interfaces:
+            intf_profile_settings = self._build_interface_with_rx_queue_settings(interface)
+            if intf_profile_settings is not None:
+                sfe_profile_member_interfaces.append(intf_profile_settings)
+
+        # iterate through member interfaces of each L3 Port-Channel
+        for interface in self.shared_utils.node_config.l3_port_channels:
+            for member_intf in interface.member_interfaces:
+                intf_profile_settings = self._build_interface_with_rx_queue_settings(member_intf)
+                if intf_profile_settings is not None:
+                    sfe_profile_member_interfaces.append(intf_profile_settings)
+
+        if sfe_profile_member_interfaces:
+            return sfe_profile_member_interfaces._natural_sorted()
+        return None
+
+    def _build_interface_with_rx_queue_settings(
+        self: AvdStructuredConfigBaseProtocol,
+        l3_member_interface: EosDesigns._DynamicKeys.DynamicNodeTypesItem.NodeTypes.NodesItem.L3InterfacesItem
+        | EosDesigns._DynamicKeys.DynamicNodeTypesItem.NodeTypes.NodesItem.L3PortChannelsItem.MemberInterfacesItem,
+    ) -> EosCliConfigGen.Platform.Sfe.Interface.ProfilesItem.InterfacesItem | None:
+        """Returns one SFE interface with profile settings for structured config."""
+        if not l3_member_interface._get("rx_queue"):
+            # specified interface does not have explicit "rx_queue" key or sub-keys specified
+            # exclude such interface from profile being built
+            return None
+        # validate rx_queue 'count' when specified
+        if l3_member_interface.rx_queue._get("count") and l3_member_interface.rx_queue.count > self.shared_utils.max_rx_queues:
+            msg = (
+                f"'rx_queue' count for interface '{l3_member_interface.name}' exceeds maximum supported '{self.shared_utils.max_rx_queues}' for this platform."
+            )
+            raise AristaAvdInvalidInputsError(msg)
+
+        rx_queue_workers = set()
+        for _worker in l3_member_interface.rx_queue.worker:
+            for worker_id in range_expand(_worker):
```

Comment:
Remove the int() in mutliple places and do it once here instead.
```suggestion
            for worker_id in map(int, range_expand(_worker)):
```

Reply:
FYI Range expand supports a list as input, and it will expand both the list and the string ranges inside recursively.